rest_api/controller/search.py

```diff
@@ -25,23 +26,10 @@ class Request(BaseModel):
 
 
 
-class Answer(BaseModel):
-    answer: Optional[str]
-    question: Optional[str]
-    score: Optional[float] = None
-    probability: Optional[float] = None
-    context: Optional[str]
-    offset_start: Optional[int]
-    offset_end: Optional[int]
-    offset_start_in_doc: Optional[int]
-    offset_end_in_doc: Optional[int]
-    document_id: Optional[str] = None
-    meta: Optional[Dict[str, Any]]
-
-
 class Response(BaseModel):
     query: str
     answers: List[Answer]
+    #maybe add: documents?
```

Comment:
Wouldn't Document also should be part of Answer? Means like user want to know if I am getting generator/summerizer output then it is based on which documents?

Reply:
Right now we have this link from answer to document via `document_id` in the answer. But you are right, this requires an extra API call if you want "extra information" from the related document. We will re-think if it makes sense to add it explicitly in the Answer itself. 

In this case here in `Response`, the intention is rather that you can call a single `query` endpoint and always get a similar response format regardless whether it's a pure QA, a pure document search or an hybrid pipeline. We had special endpoints for this in the past but it was rather confusing for users ...

Reply:
We could make it optional. User can pass extra param like `verbose` to get more data in response.